base/applications/rapps/available.cpp

```diff
@@ -18,23 +18,23 @@
 #include <atlstr.h>
 
  // CAvailableApplicationInfo
-CAvailableApplicationInfo::CAvailableApplicationInfo(const ATL::CStringW& sFileNameParam)
+CAvailableApplicationInfo::CAvailableApplicationInfo(const ATL::CStringW& sFileNameParam, AvailableStrings m_Strings)
```

Comment:
It's better to not name a function argument like we name our member variables (with the `m_` prefix)